### PR TITLE
Create Loader

### PR DESCRIPTION
**Pull request by moisesgui**

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ module.exports = {
    ** See https://axios.nuxtjs.org/options
    */
   axios: {
-    baseURL: 'https://api.kdti.dev/api/',
+    baseURL: 'https://staging.kdti.dev/api/',
```
Comment (allanhal) [inline nuxt.config.js]: That should be in another MR.

Comment (moisesgui) [reply nuxt.config.js]: ok!

File: components/JobCard.vue [hunk 2]
```diff
@@ -1,39 +1,44 @@
...
-    <h3 class="job-title">{{ job.title }}</h3>
-    <ul class="tag-list">
-      <li class="tag">R$ {{ job.salary | k }}</li>
-      <li class="tag">{{ job.seniorityLevel }}</li>
-      <li class="tag">...</li>
-    </ul>
-    <router-link :to="`/job/${job.slug}`" class="link-see-more">
-      Detalhes da vaga
-    </router-link>
+    <div v-if="false">
```
Comment (carlosxjm) [inline components/JobCard.vue]: Remove it

File: components/JobCardLoader.vue [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<script>
+import { ContentLoader } from 'vue-content-loader'
```
Comment (carlosxjm) [inline components/JobCardLoader.vue]: This lib wasn't installed with `--save`

File: pages/jobs/index.vue [hunk 4]
```diff
@@ -16,16 +15,14 @@
...
 
 export default {
   components: {
     JobCard,
-    Spinner,
   },
   data() {
     return {
-      loadingJobs: false,
+      loadingJobs: true,
```
Comment (carlosxjm) [inline pages/jobs/index.vue]: Keep it starting with `false`

File: components/JobCard.vue (hunk 2 above)
Comment (moisesgui) [reply components/JobCard.vue]: Remove or change to 'true' ?

File: components/JobCardLoader.vue (hunk 3 above)
Comment (moisesgui) [reply components/JobCardLoader.vue]: done.

File: pages/jobs/index.vue (hunk 4 above)
Comment (moisesgui) [reply pages/jobs/index.vue]: ok.

**Comment by moisesgui** [comment]
done.

File: components/JobCardLoader.vue [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+export default {
+  components: {
+    ContentLoader,
+  },
+}
+</script>
+<template>
+  <content-loader
+    :height="160"
+    :width="400"
```
Comment (carlosxjm) [inline components/JobCardLoader.vue]: Isn't it breaking the layout? This width shouldn't be fixed.

Comment (moisesgui) [reply components/JobCardLoader.vue]: when I did the tests for yarn run did'nt break ... i need to see with you later about being fixed this values

**Comment by moisesgui** [comment]
oks, will have git Rebase.

File: components/JobCardLoader.vue (hunk 5 above)
Comment (carlosxjm) [reply components/JobCardLoader.vue]: The loaders should accept % values.
Please, fix and reopen the PR. Thanks!
